test/fixtures/input.json

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
maybe call this data-resource rather than input.json as you'll probably add more going forward.